psi4/driver/procrouting/proc.py

```diff
@@ -1043,6 +1055,12 @@ def select_ccsd_at_(name, **kwargs):
         if mtd_type == 'CONV':
             if module in ['', 'MRCC'] and which("dmrcc", return_bool=True):
                 func = run_mrcc
+        elif mtd_type == "DF":
+            if module in ["OCC"]:  # SOON "",
```

Comment:
As in, soon this will be ["OCC", ""]? Why?

Reply:
This is the "semi-disabling". By not having `["", "OCC"]`, the naive input `set reference uhf; set cc_type df; energy("a-ccsd(t)")` won't run and you won't be maddened by worse-than-conventional-CC scaling. However, if you're determined to try the code, it is accessible by adding a `set qc_module occ`.